.github/workflows/inventory_updater/lib/inventory_updater.rb

```diff
@@ -138,7 +139,15 @@ def process_repository(repository)
     return serialized_cores
   end
 
-  def serialize_core(repository, core, platform, download_url, latest_release, funding)
+  def license_check(core)
+    LICENSE_SLOT_NAMES = [
+      'JTBETA' # required by jotego beta cores
+    ]
+
+    return core.data.data_slots.any? { |data_slot| LICENSE_SLOT_NAMES.include?(data_slot.name) }
+  end
+
+  def serialize_core(repository, core, platform, download_url, latest_release, funding, sponsor_only)
```

Comment:
```suggestion
  def serialize_core(repository, core, platform, download_url, latest_release, funding, requires_license)
```

Reply:
missed this one before. should be updated, now